Tests/DatadogTests/Datadog/LoggerTests.swift

```diff
@@ -9,39 +9,36 @@ import XCTest
 
 // swiftlint:disable multiline_arguments_brackets
 class LoggerTests: XCTestCase {
-    let core = DatadogCoreMock()
```

Comment:
If we make the context mutable, we could keep the `core` variable and flush everything on `tearDown`.